src/Essentials/src/AppActions/AppActions.uwp.cs

```diff
@@ -21,9 +26,19 @@ internal static bool PlatformIsSupported
 
         internal static async Task OnLaunched(LaunchActivatedEventArgs e)
         {
-            if (e?.Arguments?.StartsWith(appActionPrefix) ?? false)
+            var args = e?.Arguments;
+#if !WINDOWS_UWP
+            if (string.IsNullOrEmpty(args))
+            {
+                var cliArgs = Environment.GetCommandLineArgs();
+                if (cliArgs?.Length > 1)
+                    args = cliArgs[1];
+            }
+#endif
```

Comment:
This is for this bug: https://github.com/microsoft/microsoft-ui-xaml/issues/2005